src/app/components/card/CardButton.vue

```diff
@@ -33,18 +46,31 @@
 
 <script setup lang="ts">
 import type { RouteLocationRaw } from 'vue-router'
+import type { Component } from 'vue'
+import arrowForwardIcon from '~/assets/icons/arrow_forward.svg'
 
 export interface Props {
   description?: string
   isDisabled?: boolean
   isExternal?: boolean
   title: string
   to?: RouteLocationRaw
+  backgroundColor?: string
+  secondaryIcon?: string | Component
+  src?: string
+  alt?: string
+  textWhite?: boolean
```

Comment:
Styling like this should be set in classes on the parent item, not through JavaScript.

Reply:
Done, please review and resolve this comment yourself if you agree with my implementation.